packages/hash/integration/src/tests/model/knowledge/entity.model.test.ts

```diff
@@ -126,67 +128,74 @@ describe("Entity CRU", () => {
             destinationEntityTypeModels: ["SELF_REFERENCE"],
-            array: true,
           },
         ],
-        actorId: testUser.entityId,
+        actorId: testUser.entityUuid,
       }),
-      actorId: testUser.entityId,
+      actorId: testUser.entityUuid,
     });
   });
 
   let createdEntityModel: EntityModel;
   it("can create an entity", async () => {
     createdEntityModel = await EntityModel.create(graphApi, {
-      ownedById: testUser.entityId,
+      ownedById: testUser.entityUuid,
       properties: {
         [namePropertyTypeModel.baseUri]: "Bob",
         [favoriteBookPropertyTypeModel.baseUri]: "some text",
       },
       entityTypeModel,
-      actorId: testUser.entityId,
+      actorId: testUser.entityUuid,
     });
   });
 
   it("can read an entity", async () => {
     const fetchedEntityModel = await EntityModel.getLatest(graphApi, {
-      entityId: createdEntityModel.entityId,
+      entityId: createdEntityModel.baseId,
     });
 
-    expect(fetchedEntityModel.entityId).toEqual(createdEntityModel.entityId);
+    expect(fetchedEntityModel.baseId).toEqual(createdEntityModel.baseId);
     expect(fetchedEntityModel.version).toEqual(createdEntityModel.version);
   });
 
   let updatedEntityModel: EntityModel;
   it("can update an entity", async () => {
-    expect(createdEntityModel.createdById).toBe(testUser.entityId);
-    expect(createdEntityModel.updatedById).toBe(testUser.entityId);
+    expect(createdEntityModel.metadata.provenance.createdById).toBe(
+      testUser.entityUuid,
+    );
+    expect(createdEntityModel.metadata.provenance.updatedById).toBe(
+      testUser.entityUuid,
+    );
 
     updatedEntityModel = await createdEntityModel
       .update(graphApi, {
         properties: {
           [namePropertyTypeModel.baseUri]: "Updated Bob",
           [favoriteBookPropertyTypeModel.baseUri]: "Even more text than before",
         },
-        actorId: testUser2.entityId,
+        actorId: testUser2.entityUuid,
       })
       .catch((err) => Promise.reject(err.data));
 
-    expect(updatedEntityModel.createdById).toBe(testUser.entityId);
-    expect(updatedEntityModel.updatedById).toBe(testUser2.entityId);
+    expect(updatedEntityModel.metadata.provenance.createdById).toBe(
+      testUser.entityUuid,
+    );
+    expect(updatedEntityModel.metadata.provenance.updatedById).toBe(
+      testUser2.entityUuid,
+    );
   });
 
   it("can read all latest entities", async () => {
     const allEntityModels = (
       await EntityModel.getByQuery(graphApi, {
         all: [{ equal: [{ path: ["version"] }, { parameter: "latest" }] }],
       })
-    ).filter((entity) => entity.ownedById === testUser.entityId);
+    ).filter((entity) => entity.ownedById === testUser.entityUuid);
 
     const newlyUpdatedModel = allEntityModels.find(
-      (ent) => ent.entityId === updatedEntityModel.entityId,
+      (ent) => ent.entityUuid === updatedEntityModel.entityUuid,
```

Comment:
I think in places like this we do want to be doing `baseId`

Reply:
Sure, done in https://github.com/hashintel/hash/pull/1435/commits/19a4db59d2a19165604db0ab3dd99b964535c3fd